pkgs/by-name/so/soundconverter/package.nix

```
@@ -0,0 +1,116 @@
+{
+  lib,
+  fetchurl,
+  # Optional due to unfree license.
+  faacSupport ? false,
+  glib,
+  python3Packages,
+  gtk3,
+  wrapGAppsHook,
+  gsettings-desktop-schemas,
+  intltool,
+  xvfb-run,
+  gobject-introspection,
+  gst_all_1,
+  fdk-aac-encoder,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "soundconverter";
+  version = "4.0.5";
+
+  src = fetchurl {
+    url = "https://launchpad.net/soundconverter/trunk/${version}/+download/${pname}-${version}.tar.gz";
+    hash = "sha256-ebBaNmhd3OXeJPCiPQjgCgFO2J5ilx/pTB0/uDbrefM=";
+  };
+
+  buildInputs = [
+    gtk3
+    fdk-aac-encoder
+    gst_all_1.gst-libav
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    (gst_all_1.gst-plugins-bad.override { inherit faacSupport; })
+  ];
+
+  nativeBuildInputs = [
+    intltool
+    wrapGAppsHook
+    gobject-introspection
+  ];
+
+  propagatedBuildInputs = [
+    python3Packages.gst-python
+    python3Packages.distutils-extra
+    python3Packages.setuptools
+    python3Packages.pygobject3
+  ];
+
+  nativeCheckInputs = [ xvfb-run ];
+
+  postPatch = ''
+    substituteInPlace  bin/soundconverter --replace-warn \
```

```suggestion
    substituteInPlace bin/soundconverter --replace-fail \
```
we want to hard fail if that doesn't match anymore

---

Fixed